setup.cfg

```diff
@@ -47,7 +47,13 @@ install_requires =
     napari-tools-menu
     napari-skimage-regionprops>=0.3.1
     hdbscan
-    joblib
+    qtpy
+    dask-core
```

Comment:
I think this is breaking the tests, maybe we can just put the whole of dask as a dependency? One problem I see is that the pypi installation for the complete dask library requires an import that looks like this: pip install "dask[complete]"  [see documentation](https://docs.dask.org/en/stable/install.html). I do not know how to specify this in requirements for conda and pip seperately though since they have differing commands. Maybe we should check how it is done i napari..

Reply:
In napari they seem to mention dask in the requirements like this:

    dask[array]>=2.15.0,!=2.28.0  # https://github.com/napari/napari/issues/1656
[see here](https://github.com/napari/napari/blob/6affa18d1d5782bb31532c5f6be07f4a31e9a503/setup.cfg#L50) but I think that might change at some point because their weird version pinning seems to be related to an issue with a dask update so maybe we should not specify the exact version and then the dependency solver will take care of it?

Reply:
@Cryaaa I think I am back to the previous error which is also failing on the main branch ^^" Thanks for spotting!